tools/hybrid-quickstart/hybrid13/example-proxy/apiproxy/proxies/default.xml

```diff
@@ -16,6 +16,15 @@
 -->
 
 <ProxyEndpoint name="default">
```

Comment:
why?

Reply:
Because this way it shows the flows in the UI and makes for an easier demo.